.github/workflows/build.yml

```
@@ -63,7 +63,7 @@ jobs:
           python -c 'import mmcv; print(mmcv.__version__)'
       - name: Install unittest dependencies
         run: |
-          pip install -r requirements.txt
+          pip install -r requirements.txt -r requirements/albu.txt
```

```suggestion
          pip install -r requirements.txt
```
As the albu requirement is moving to optional, no need to add here, similarly hereinafter